buddyboss/buddyboss-platform
# Embeds > Fix Preview of embeds on site

### All Submissions:






### Changes proposed in this Pull Request:



Closes #383 
https://github.com/buddyboss/buddyboss-theme/issues/293

### How to test the changes in this Pull Request:

1. Go to activity post form
2. paste any url which is embeddable eg. https://techcrunch.com/2019/12/18/watch-a-full-test-flight-of-liliums-all-electric-urban-aircraft/
3. Save activity with preview
4. See previews of link

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Activity link embed and previews fixed
